deployConfig/poolConfig.ts

```diff
@@ -661,27 +661,27 @@ export const poolConfigs: { [name: string]: poolConfig } = {
     },
   },
 
-  aUSDC_v8: {
```

Comment:
why are we removing aUSDC_v8?

Reply:
[7a8ac37](https://github.com/Voltz-Protocol/voltz-core/pull/229/commits/7a8ac379b3c04ccaa0cc7c9481e4ad033bdf35c2)